apps/framework-cli/src/cli.rs

```diff
@@ -200,51 +219,6 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
                 controller.add_routine(Box::new(CleanProject::new(project_arc, run_mode)));
                 controller.run_routines(run_mode);
             }
-            Commands::Docker { sub_method } => match sub_method.as_str() {
-                "init" => {
-                    let run_mode = RunMode::Explicit {};
-                    info!("Running docker init command");
-                    let project = Project::load_from_current_dir()
-                        .expect("No project found, please run `moose init` to create a project");
-                    let project_arc = Arc::new(project);
-
-                    crate::utilities::capture::capture!(
```

Comment:
Looks like we are loosing this with the change - we probably want to instrument the `build` command all the same

Reply:
Added new capture points. I also refactored capture.rs ActivityType's to match this PRs changes.